spark/v3.3/spark/src/main/java/org/apache/iceberg/spark/SparkReadConf.java

```diff
@@ -132,14 +134,34 @@ public boolean streamingSkipDeleteSnapshots() {
         .parse();
   }
 
-  public boolean streamingSkipOverwriteSnapshots() {
+  private boolean streamingSkipOverwriteSnapshots() {
```

Comment:
why did you change the visibility of this method? I think it's a blocker for a revision to change the public interface

Reply:
I intend to deprecate the old conf and just keep the new one, anyway this was the only usage of the method